go.mod

```diff
@@ -70,5 +75,6 @@ replace (
 	github.com/Azure/go-autorest/autorest/adal => github.com/Azure/go-autorest/autorest/adal v0.6.0
 	github.com/Azure/go-autorest/autorest/azure/auth => github.com/Azure/go-autorest/autorest/azure/auth v0.3.0
 	github.com/golang/glog => github.com/kubermatic/glog-logrus v0.0.0-20180829085450-3fa5b9870d1d
+	google.golang.org/grpc => google.golang.org/grpc v1.26.0
```

Comment:
We are trying to avoid these. Do you know where the conflict arises?

Reply:
Problem stems from backwards incompatible changes to gRPC that most dependencies have upgraded past yet, resulting in errors such as these:
```
# github.com/coreos/etcd/clientv3/balancer/resolver/endpoint
../../go/pkg/mod/github.com/coreos/etcd@v3.3.22+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:114:78: undefined: resolver.BuildOption
../../go/pkg/mod/github.com/coreos/etcd@v3.3.22+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:182:31: undefined: resolver.ResolveNowOption
github.com/coreos/etcd/pkg/types
github.com/coreos/etcd/clientv3/balancer/connectivity
github.com/coreos/etcd/clientv3/balancer/picker
github.com/coreos/etcd/version
# github.com/coreos/etcd/clientv3/balancer/picker
../../go/pkg/mod/github.com/coreos/etcd@v3.3.22+incompatible/clientv3/balancer/picker/err.go:37:44: undefined: balancer.PickOptions
../../go/pkg/mod/github.com/coreos/etcd@v3.3.22+incompatible/clientv3/balancer/picker/roundrobin_balanced.go:55:54: undefined: balancer.PickOptions
```
 
Unfortunately the very same commit of the Istio client that upgrades to v1.18.x also bumps gRPC to v1.28 causing misery unless pinned back so the rest of the code continues to function.

Reply:
Upstream issue in etcd regarding the API change: https://github.com/etcd-io/etcd/issues/11563

Reply:
It might be possible to pin the etcd client instead but I haven't tried that as it has a larger surface area and imported via more than one package path due to change from coreos org. Regardless at least one pin will be required to get Contour, Istio and Linode dependencies to play nice it's just a matter of which one.

Reply:
Got it. Can we add a TODO comment, just so that we can get back to it at a later time?